app/entry.server.tsx

```diff
@@ -1,4 +1,5 @@
 import { PassThrough } from 'stream'
+import { styleText } from 'node:util'
```

Comment:
I am planning on a future PR that makes sure all builtin imports are prefixed with `node:`, so the ESLint warning for import order should be fine (unless you want it fixed

Reply:
Let's fix it